UI/app/src/main/java/com/example/pathfinder/Activities/DriverDashboard.java

```diff
@@ -44,15 +45,15 @@
     private static final int IMAGE_WIDTH = 320;
     private static final int IMAGE_HEIGHT = 240;
 
+    //key names for stored data in shared preferences
+    private static final String KEY_STOP = "stop";
+    private static final String KEY_HANDICAP = "handicap";
+
     private MqttClient mMqttClient;
-    //Park/ lock
-    //private boolean isParked = false;
-    //Engine activity
-    //private boolean isActive = false;
     private boolean isConnected = false;
-    private ImageView mVideoStream, mSignOutBtn;
+    private ImageView mVideoStream, mSignOutBtn, mAccessibilityRequest;
     private TextView mSpeedLog, mDistanceLog;
-    private TextView textView;
+    private TextView textView, mStopRequest;
```

Comment:
Is this a way to declare 2 different variables in one line ? If so, that is pretty cool :p

Reply:
Yep! Now that you mention it, I could probably combine some others for better consistency :)

Reply:
Cool, I did not know we could do that. Good to know 😄